openjdk-integ-tests/src/test/java/libcore/javax/net/ssl/SSLSocketTest.java

```diff
@@ -1649,7 +1658,8 @@ private void test_SSLSocket_interrupt_case(boolean readUnderlying, boolean close
             final InputStream inputStream = toRead.getInputStream();
             @SuppressWarnings("unused")
             int value = inputStream.read();
-            fail();
+            // FD-based socket on OpenJDK seems to return -1 here.
+            assertEquals(-1, value);
```

Comment:
Actually this didn't seem to be the case. This has been reverted since it wasn't needed.